documentation/release_notes.rst

```diff
@@ -57,8 +57,11 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
-- Incorrect results may be produced by ``reduce`` and ``transform_reduce`` with 64-bit types and ``std::multiplies``,
-  ``sycl::multiplies`` operations when compiled by Intel® C++ Compiler 2021.3 and newer and executed on GPU devices. 
+- Incorrect results may be produced by ``reduce``, ``reduce_by_segment`` and ``transform_reduce``
+  with 64-bit data types when compiled by Intel oneAPI DPC++/C++ Compiler 2021.3 and newer
+  and executed on GPU devices.
+  For a workaround, define the ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION macro to 1
+  before including oneDPL header files.
```

Comment:
Oops, I missed that this change is for the description in the "old" 2022.3 release, where the workaround is not available. Let's remove the last sentence; we will need to describe it in 2022.5 release notes. Sorry for confusion.

Reply:
Fixed.